### PR TITLE
feat: [sc-16700] Update test-app job

### PR DESCRIPTION
**Pull request by Cwagne17**
Story details: https://app.shortcut.com/clarkcan/story/16700

### CODE REVIEW

**Review by JosueMolinaMorales — APPROVED** [review]

File: src/jobs/test.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    description: |
+      The language the project is using.
+
+  # ---------------------------------------------------------------------------------
+  # OPTIONAL PARAMETERS
+  # These parameters have reasonable defaults.
+  # ---------------------------------------------------------------------------------
+  resource-class:
+    type: string
+    default: "medium"
```
Comment (pzalep1) [inline src/jobs/test.yml:21]: What are the other options for this?

**Review by pzalep1 — COMMENTED** [review]
Just the one question

File: src/jobs/test.yml (hunk 1 above)
Comment (Cwagne17) [reply src/jobs/test.yml]: I just added. The performance plan that we have supports:
Screen Shot 2022-12-30 at 1 04 11 PM

**Review by pmanal1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by AStoerkel1 — APPROVED** [review]

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.3.12
+2.0.0-alpha
```
Comment (mfranz1) [inline VERSION:1]: We want to version this to alpha?

**Review by mfranz1 — APPROVED** [review]

File: VERSION (hunk 2 above)
Comment (Cwagne17) [reply VERSION:1]: I was just thinking once we know it is working we can put it to 2.0.0

**Pull request merged**